src/components/term_row.js

```diff
@@ -1,5 +1,5 @@
-import ansi_up from 'ansi_up';
 import React from 'react';
+import aha from '../utils/aha';
```

Comment:
change to `import aha from 'aha';` which instructs node to fetch from the local dependencies downloaded from npm, declared in the package.json